zebra-consensus/src/block.rs

```diff
@@ -206,13 +206,16 @@ where
             let new_outputs = Arc::try_unwrap(known_utxos)
                 .expect("all verification tasks using known_utxos are complete");
 
+            let block_utxos = transparent::utxos_from_ordered_utxos(new_outputs.clone());
```

Comment:
We start with the utxos in the new_outputs as this is what we have available at this moment and we need to fill the field of the `PreparedBlock`.